Cython/Compiler/ParseTreeTransforms.py

```diff
@@ -2557,6 +2560,24 @@ def visit_CClassDefNode(self, node):
         self.in_py_class = old_in_pyclass
         return node
 
+    def _body_without_directives(self, node, directives_to_remove):
```

Comment:
The name sounds like it returns something, but instead, it _does_ something. So there should be a verb in here.